app/views/inventories/index.html.erb

```diff
@@ -27,7 +27,13 @@
       <% @inventories.each do |inventory| %>
         <tr>
           <td class="text-center"><%= inventory.id %></td>
-          <td>Thumnail here</td>
+          <td>
+            <% if inventory.images.attached? %>
+              <%= image_tag create_thumbnail(inventory), class: "img-thumbnail" %>
+            <% else %>
+              Thumbnail here
```

Comment:
Should use a default thumbnail for this case.

Reply:
You mean a default image replace "Thumbnail here"?